android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackService.kt

```diff
@@ -0,0 +1,152 @@
+package com.brentvatne.exoplayer
+
+import android.annotation.SuppressLint
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.content.Context
+import android.content.Intent
+import android.os.Binder
+import android.os.Build
+import android.os.Bundle
+import android.os.IBinder
+import androidx.core.app.NotificationCompat
+import androidx.media3.exoplayer.ExoPlayer
+import androidx.media3.session.CommandButton
+import androidx.media3.session.MediaSession
+import androidx.media3.session.MediaSessionService
+import androidx.media3.session.MediaStyleNotificationHelper
+import androidx.media3.session.SessionCommand
+import okhttp3.internal.immutableListOf
+
+class PlaybackServiceBinder(val service: VideoPlaybackService) : Binder()
+
+class VideoPlaybackService : MediaSessionService() {
+    private var mediaSessionsList = mutableMapOf<ExoPlayer, MediaSession>()
+    private var binder = PlaybackServiceBinder(this)
+
+    // Controls
+    private val commandSeekForward = SessionCommand(COMMAND_SEEK_FORWARD, Bundle.EMPTY)
+    private val commandSeekBackward = SessionCommand(COMMAND_SEEK_BACKWARD, Bundle.EMPTY)
+
+    @SuppressLint("PrivateResource")
+    private val seekForwardBtn = CommandButton.Builder()
+        .setDisplayName("forward")
+        .setSessionCommand(commandSeekForward)
+        .setIconResId(androidx.media3.ui.R.drawable.exo_notification_fastforward)
+        .build()
+
+    @SuppressLint("PrivateResource")
+    private val seekBackwardBtn = CommandButton.Builder()
+        .setDisplayName("forward")
+        .setSessionCommand(commandSeekBackward)
+        .setIconResId(androidx.media3.ui.R.drawable.exo_notification_rewind)
+        .build()
+
+    // Player Registry
+
+    fun registerPlayer(player: ExoPlayer) {
+        if (mediaSessionsList.containsKey(player)) {
+            return
+        }
+
+        val mediaSession = MediaSession.Builder(this, player)
+            .setId("RNVideoPlaybackService_" + player.hashCode())
+            .setCallback(VideoPlaybackCallback(10000L))
```

Comment:
maybe extract this 10 sec to a constant and add a comment indicating this is hardcoded. 10sec is OK for me